src/components/NcTextField/NcTextField.vue

```diff
@@ -51,8 +51,10 @@ and `minlength`.
 			@trailing-button-click="clearText">
 		</NcTextField>
 		<NcTextField label="Disabled"
+			value="''"
```

Comment:
Is the content of the literal string `''` expected, or did you mean:
```suggestion
			:value="''"
```

Reply:
What you wrote, I made a typo 😬

Reply:
replaced with existing data attrs